src/roswire/ros2/launch.py

```diff
@@ -131,7 +109,10 @@ def launch(self,
                args: Optional[Mapping[str, Union[int, str]]] = None,
                prefix: Optional[str] = None,
                launch_prefixes: Optional[Mapping[str, str]] = None,
-               node_to_remappings: Optional[Mapping[str, Collection[Tuple[str, str]]]] = None  # noqa
+               node_to_remappings: Optional[
+                   Mapping[str, Collection[Tuple[str, str]]]
+               ] = None
+               # noqa
```

Comment:
`noqa` is no longer needed since this has been broken across multiple lines

Reply:
Done

Reply:
It looks like `#noqa` is still on line 115?